### PR TITLE
Continue to support ebs provisioning

### PR DESCRIPTION
**Pull request by bobotimi**

### CODE REVIEW

**Review by jbermudezsonatype — APPROVED** [review]

File: nxrm-ha/README.md [hunk 1]
```diff
@@ -25,6 +25,42 @@ This Helm chart configures the Kubernetes resources that are needed for a high-a
 The default configuration uses an emptyDir volume for storing Nexus Repository logs. However, this is only for demonstration purposes. For production, we strongly recommend that
 you configure dynamic provisioning of persistent storage bound to a shared location, such as EFS/Azure File/NFS, which is accessible to all actives nodes in your Kubernetes cluster. 
 
+Note:
+- The supported storage mechanism up to version 66.0.0 of this helm chart was local storage such as EBS/Azure Disk/locally attached disks (for onprem deployments)
+- From version 68.0.0 onwards, the recommended storage mechanism is shared storage such as EFS/Azure File/NFS (for onprem deployments). However, the chart is still compatible with local storage.
```
Comment (lisadurant) [inline nxrm-ha/README.md:33]:
```suggestion
> **_Note:_**  Versions **66.0.0 and older** of this chart only supported local storage (e.g., EBS, Azure Disk, locally attached disks for on-prem deployments). 
> 
> From version **68.0.0+**, we recommend and support using **shared storage** (e.g., EFS, Azure File, NFS for on-prem deployments). However, this chart is still compatible with local storage.
```

Comment (lisadurant) [reply nxrm-ha/README.md:33]: ^ I did test this on a markdown live preview, and it should all still be contained in the "note" box fwiw.

File: nxrm-ha/README.md [hunk 2]
```diff
@@ -25,6 +25,42 @@ This Helm chart configures the Kubernetes resources that are needed for a high-a
...
+
+
+#### Migrating from local storage (i.e. EBS/Azure Disk/local disk) to EFS/Azure File/NFS storage for Nexus Repository logs
+If you have version 66.0.0 (or lower) of the nxrm-ha chart installed and wish to switch to using shared storage for Nexus Repository logs from , please do as follows:
```
Comment (lisadurant) [inline nxrm-ha/README.md:37]:
```suggestion
#### Migrating from local storage (i.e., EBS/Azure Disk/local disk) to shared storage (i.e., EFS/Azure File/NFS) for storing Nexus Repository logs

If you have installed version 66.0.0 or older of the nxrm-ha chart and wish to switch to using shared storage for Nexus Repository logs, please do as follows:
```

File: nxrm-ha/README.md [hunk 3]
```diff
@@ -25,6 +25,42 @@ This Helm chart configures the Kubernetes resources that are needed for a high-a
...
+  - Confirm new PVCs were created and that they are bound appropriately depending on your configuration (i.e. EFS access points/Azure File shares/NFS mounts)
+  - kubectl get pvc -n nexusrepo 
+  - kubectl describe pvc -n nexusrepo nexus-data-nxrm-nxrm-ha-0 
+  - kubectl describe pvc -n nexusrepo nexus-data-nxrm-nxrm-ha-1 
+  - Check that the 'volume.beta.kubernetes.io/storage-provisioner' annotation in the PVC description is representative of your configuration i.e. EFS/Azure File/NFS provisioner
+  - Confirm pods are up and running:
+    - kubectl get pods -n nexusrepo 
+    - kubectl logs -n nexusrepo nxrm-nxrm-ha-0 -f 
+    - kubectl logs -n nexusrepo nxrm-nxrm-ha-1 -f 
+    - kubectl logs -n nexusrepo nxrm-nxrm-ha-2 -f
```
Comment (lisadurant) [inline nxrm-ha/README.md:65]:
```suggestion
1. Back up all of your Nexus Repository pods' logs by logging in and generating a support zip for each one (see our [support zip help documentation](https://help.sonatype.com/en/support-features.html#creating-a-support-zip-in-a-high-availability-environment) to learn how).
2. Scale statefulset replicas to zero using commands like the following:
   * `kubectl get statefulsets -n nexusrepo`
   * `kubectl scale statefulsets -n nexusrepo <stateful set name> --replicas=0`
3. After scaling statefulset replicas to zero, delete the statefulset using a command like the following:
   *`kubectl delete statefulsets -n nexusrepo <stateful set name>`
4. Delete EBS/Azure disk/local disk PVCs using commands like the following: 
   * `kubectl get pvc -n nexusrepo`
   * `kubectl delete pvc -n nexusrepo <pvc 1> <pvc 2> <pvc n>`
   * If you have a custom storage class for EBS PVC/PV pair, delete that storage class as well.
4. Upgrade to the new version (68.0.0) of the nxrm-ha helm chart by taking the following steps:
   * Provision your shared storage (e.g., EFS/Azure File/NFS), and set appropriate permissions.
   * Update your custom values.yaml file for shared storage as detailed in the applicable section below for configuring dynamic persistent volume provisioning for your selected storage option. 
   * Run helm upgrade: `helm upgrade nxrm sonatype/nxrm-ha -f <your values.yaml> --version 68.0.0`
   * Scale replicas as needed.
5. Confirm all Nexus Repository pods start up and create new PVCs.
   * Confirm new PVCs were created and that they are bound appropriately depending on your configuration (i.e., EFS access points/Azure File shares/NFS mounts):
      * `kubectl get pvc -n nexusrepo` 
      * `kubectl describe pvc -n nexusrepo nexus-data-nxrm-nxrm-ha-0` 
      * `kubectl describe pvc -n nexusrepo nexus-data-nxrm-nxrm-ha-1`
      * Check that the `volume.beta.kubernetes.io/storage-provisioner` annotation in the PVC description is representative of your configuration (i.e., EFS/Azure File/NFS provisioner).
   * Confirm pods are up and running using commands like the following:
      * `kubectl get pods -n nexusrepo` 
      * `kubectl logs -n nexusrepo nxrm-nxrm-ha-0 -f` 
      * `kubectl logs -n nexusrepo nxrm-nxrm-ha-1 -f` 
      * `kubectl logs -n nexusrepo nxrm-nxrm-ha-2 -f`
```

Comment (lisadurant) [reply nxrm-ha/README.md:65]: ^ Ran this through markdown preview as well

**Review by lisadurant — CHANGES_REQUESTED** [review]

**Review by lisadurant — APPROVED** [review]

**Pull request merged**